pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -90,9 +90,15 @@ type watcher struct {
 
 type eventWatcherCache struct {
 	HeadCommit string
+	GitPath    string
 	Configs    []config.EventWatcherConfig
 }
 
+type eventWatcherConfig struct {
+	GitPath string
```

Comment:
Why do we need `GitPath` in both places?

Reply:
Because we need to generate the file path from the file name and the git path.
https://github.com/pipe-cd/pipecd/pull/3757/files#diff-60908ffb7e37bfc3692d8c4cace4be91d53bccf9a549fb91ea30b70387abd9cfR598-R603